apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -206,15 +206,17 @@ export default function BaseMegaMenu() {
                         <hr className="mb-3.5" />
                         <ul>
                           {items.map((item) => (
-                            <SfListItem size="sm" role="none">
-                              <a
-                                key={item.title}
-                                className="focus-visible:outline focus-visible:rounded-sm"
+                            <li key={item.title}>
+                              <SfListItem
+                                as="a"
+                                size="sm"
+                                role="none"
                                 href={item.link}
+                                className="focus-visible:outline focus-visible:rounded-sm"
                               >
                                 {item.title}
-                              </a>
-                            </SfListItem>
+                              </SfListItem>
+                            </li>
```

Comment:
Make sure hover background area covers with the focus ring.I believe it should be resolved when you adjust to the latest changes of the ListItem.
<img width="176" alt="Screenshot 2023-04-21 at 12 00 21" src="https://user-images.githubusercontent.com/34419118/233608932-19dc3882-0379-497f-8d3e-97c27fbb7d0a.png">